### PR TITLE
[New] Add query-string@5.0.1 w/ git auto-update

### PR DESCRIPTION
**Pull request by kirintwn**
Pull request for issue: #10334

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/sindresorhus/query-string
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/query-string
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 22
   - Count of stars: 1662
   - Count of forks: 144
 * NPM download stats (optional):
   - Downloads in the last day: 365,829
   - Downloads in the last week: 1,972,824
   - Downloads in the last month: 8,347,989

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/query-string/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "url": "https://github.com/SaneMethod/sindresorhus/query-string.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/SaneMethod/sindresorhus/query-string.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "*.(js|map)"
```
Comment (sufuf3) [inline ajax/libs/query-string/package.json]: This pattern can't get any files.
Would you please revise this after we got the answer from upstream (which file can be used in web front-end)?
Many thanks

Comment (sashberd) [reply ajax/libs/query-string/package.json]: @sufuf3 What do you mean by get answer from upstream? There is no way to ask author about file using cause there is not issues tab in lib git repo.
@sindresorhus As I saw you are the author of this lib. Could you help us to know which files can be used in web frontend?

File: ajax/libs/query-string/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "query-string",
+  "filename": "index.min.js",
```
Comment (MattIPv4) [inline ajax/libs/query-string/package.json]:
```suggestion
  "filename": "index.js",
```
Minified file doesn't not exist in the package or auto-update config.

File: ajax/libs/query-string/package.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SaneMethod/sindresorhus/query-string.git"
+  },
+  "npmName": "query-string",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "index.js"
```
Comment (MattIPv4) [inline ajax/libs/query-string/package.json]:
```suggestion
        "index.*"
```
We should also include the typescript typings for those that may want it.

**Pull request merged**